src/js/common/schemaform/validation.js

```diff
@@ -95,9 +98,9 @@ export function uiSchemaValidate(errors, uiSchema, schema, formData, formContext
             }
           };
         }
-        uiSchemaValidate(errors, uiSchema.items, schema.items, formData, formContext, newPath);
+        uiSchemaValidate(errors, uiSchema.items, schemaWithDefinitions.items, definitions, formData, formContext, newPath);
       });
-    } else {
+    } else if (!uiSchema.items) {
       Object.keys(uiSchema)
         .filter(prop => !prop.startsWith('ui:'))
         .forEach((item) => {
```

Comment:
When submitting the application, I get the following error at `_.get(path, errors)[item] = {`:

```
Uncaught TypeError: Cannot set property 'veteranFullName' of undefined
```

`path` is an empty string and `errors` is an empty object, so it fails. Not sure what the root problem is, though.